mlir/include/mlir/Analysis/Presburger/QuasiPolynomial.h

```diff
@@ -39,6 +39,8 @@ class QuasiPolynomial : public PresburgerSpace {
   QuasiPolynomial(unsigned numVars, SmallVector<Fraction> coeffs = {},
                   std::vector<std::vector<SmallVector<Fraction>>> aff = {});
```

Comment:
not for this patch but whenever there's an const input vector parameter you should make it arrayref